### ISSUE TITLE
constants error

### ISSUE THREAD

**Issue by rewiaca** [occrp/cronosparser#4]
```
cronosparser-0.2-py3.7.egg\cronos\constants.py", line 37, in <module>
    PK_SENTINEL = chr(len(PK_SENTINEL)) + PK_SENTINEL
TypeError: can only concatenate str (not "bytes") to str
```

Fixed this but there are multiple another error followed me.
Launched by:

```
cronos.parse(
    'C:\\cronosparser-master\\test_data\long_text',
    'C:\\cronosparser-master\\test_data\long_text'
)
```

Any update or hint?

**Comment by pudo** [comment]
I think `cronosparser` would require a significant amount of work to run under Python 3...

**Comment by pudo** [comment]
Fixed in #5.
